addons/project/models/project_project.py

```diff
@@ -996,3 +999,11 @@ def _add_followers(self, partners):
                     dict_partner_ids_to_subscribe_per_partner[task.partner_id] = partner_ids_to_subscribe
         for partner, tasks in dict_tasks_per_partner.items():
             tasks.message_subscribe(dict_partner_ids_to_subscribe_per_partner[partner])
+
+    @api.depends('task_count', 'closed_task_count')
+    def _compute_task_completion_percentage(self):
+        for tasks in self:
+            if tasks.task_count > 0:
+                tasks.task_completion_percentage = tasks.closed_task_count / tasks.task_count
+            else:
+                tasks.task_completion_percentage = 0
```

Comment:
```suggestion
        for task in self:
            task.task_completion_percentage = task.task_count and 1 - task.open_task_count / task.task_count
```
- `task` instead of `tasks`
- `and` instead of `if/else` blocks
- `1 - open/total` instead of `closed/total` => no need for a new field